parachain/src/command.rs

```diff
@@ -111,6 +107,19 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, St
         )),
         "moonbase-alpha" => Box::new(chain_spec::testnet_kintsugi::staging_testnet_config(1002.into())),
         path => {
+            if let Some(matches) = Regex::new(r"^rococo-local-([0-9]+)$").unwrap().captures(path) {
```

Comment:
```suggestion
            if let Some(matches) = Regex::new(r"^rococo-local-kintsugi-([0-9]+)$").unwrap().captures(path) {
```

Reply:
This would break existing usage which is not worth it imo